include/RAJA/config.hpp.in

```diff
@@ -156,9 +156,9 @@ namespace RAJA {
 #endif // RAJA_ENABLE_OPENMP
 
 #if defined(RAJA_ENABLE_CUDA)
-#if not defined(__CUDACC__)
-#error RAJA configured with ENABLE_CUDA, but CUDA not supported by current compiler 
-#endif // 
+//#if not defined(__CUDACC__)
+//#error RAJA configured with ENABLE_CUDA, but CUDA not supported by current compiler
+//#endif //
```

Comment:
@davidbeckingsale , @rhornung67 is the error above necessary? We have some packages who use RAJA but not RAJA-CUDA (and are not compiled with nvcc), and some packages who do use RAJA-CUDA (and are compiled with nvcc).